src/doc/dev_manual/LiveIncidentResponse.rst

```diff
@@ -0,0 +1,121 @@
+Live Incident Response
+======================
+
+VisIt_ software developers also have to serve in customer support roles taking
+hotline calls, replying to emails, responding to drop-ins, etc. Customer support
+is about the *operations* side of VisIt_ software. That is the side associated
+with users *running* the software to get their work done. Apart from the VisIt_
+software itself, *operations* involves many other things including the computing
+hardware, file systems and networks upon which VisIt_ software runs as well as
+the security enviornment and any third party software dependencies involved.
+
+Most of those aspects of *operations* are typically out of the purview of the
+VisIt_ team. In most cases the extent of the VisIt_ team's involvement in
+*operations* is confined primarly to the VisIt_ software itself; its
+development, testing, release and deployment which includes installations we
+directly support on systems where we have accounts, hosted binary downloads for
+common platforms and the tools to build from sources.
+
+The practice of having software development staff *integrated* with *operations*
+is more commonly referred to as *DevOps*. There is a pretty good
+`video <https://youtu.be/XoXeHdN2Ayc>`_ that introduces these concepts.
+
+This document explains our procedures for how VisIt_ software developers handle
+active customer support calls. Some of the goals of this proccess are...
+
+  * To try to *load balance* the work of incident response in an equitable way
+    across the development team.
+  * To achieve a 1-hour response time.
+  * To reduce incident response interruptions for the team as a whole.
+
```

Comment:
Perhaps an additional goal could be something like: to appropriately escalate difficult tasks